### ISSUE TITLE
[a11y-defect-1]: Veterans experiencing homelessness cannot remove a prior mailing address from their account.

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#61364]
### Point of contact

joshkimux

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

If someone denotes they are experiencing homelessness, they should be able to remove their existing mailing address within the flow. 

If it isn't technically feasible, we should explain to the user why. 

I reached out to @adamwhitlock1 , who works on profile, for more context:
> "We technically can't remove the main mailing address once one is present and can only edit it. I wonder if there could be something added to indicate homelessness that would 'ignore' the current mailing address or indicate not to send mail there."

### Reproduction steps

1. Log in to NoD as user 236
2. On the homelessness question page, click "yes"
3. On the contact information page, edit mailing address
4. Confirm there is no way to delete an existing mailing address

### Proposed solution or next steps

Based on @adamwhitlock1 's response, it's not clear yet what the best next step might be to take. I hypothesize this could use some discovery work to determine:
- Is there a need for Veterans experiencing homelessness to remove a prior mailing address
  - e.g. are there any cases related to sensitive personal information being sent to a previous address? How can Veterans remove a previous address to avoid that? Could we reach out to someone like Karen Cutright with more experience working with Veterans experiencing homelessness for more information?
- If it is a need, we should determine:
  - Why we can't remove a main mailing address when it is added
  - If there are technical reasons we can't do this that we can't control, what is the next best step from a design/content perspective to handle this in a Veteran-centered way?

I recommend assigning a designer to look into this.

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [X] Something else

**Comment by eileen-coforma** [comment]
This should be assigned to the profile team, @Mottie will do.

**Comment by davidakennedy** [comment]
I removed the accessibility-related labels on this. After talking about this ticket in our Team Share meeting, it's not really an accessibility ticket, but more of a UX/happy path one.

**Comment by eileen-coforma** [comment]
Removed it from the a11y epic and added to design debt epic
